docs/getting-started-locally.md

```diff
@@ -0,0 +1,93 @@
+# Starting an Interledger Node Locally Using moneyd
+
+This tutorial describes how to:
+1. Install [`moneyd`](../concepts/moneyd.md) on your system
+2. Start an [Interledger](https://github.com/interledgerjs/ilp-connector) node on a local test network using moneyd
+3. Send and receive payments using the [SPSP](../concepts/spsp.md) (Simple Payment Setup Protocol) API
+
+This tutorial is useful if you’re doing local development. 
+
+![Getting-Started-Locally](../assets/getting-started-locally.png)
+
+## Before you begin
+* Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
+* Set up Node to install modules globally <br/>
+Open a command line and use the following commands:  
```

Comment:
Is this necessary?

Reply:
I had to do this when I was setting up node.js on my mac; I will remove it, since this is node.js specific unless developers would like to see it.